backend/dal/Migrations/v01.06.01/Up/PreUp/01-NotificationTemplates.sql

```diff
@@ -0,0 +1,742 @@
+UPDATE dbo.[NotificationTemplates]
+SET [Body] = '
+@using System.Linq
+@using Pims.Dal.Entities
+
+<html><head><title>@Model.Environment.Title</title></head><body><p>Dear @Model.ToAgency.AddressTo,</p><p>Your project @Model.Project.ProjectNumber has been approved. Signin to <a href="@Model.Environment.Uri">PIMS</a> to review the progress.</p><p>@Model.Project.Notes.FirstOrDefault(n => n.NoteType == NoteTypes.ErpNotification)?.Note</p><p>Sincerely Real Property Division</p></body></html>'
+WHERE [Id] = 4
+
+UPDATE dbo.[NotificationTemplates]
+SET [Body] = '
+@using System.Linq
+@using Pims.Dal.Entities
+@using System.Globalization
+
+<html><head><title>@Model.Environment.Title</title></head><body>
+<p>Good afternoon,</p>
+<p>As detailed in the <a href="https://intranet.gov.bc.ca/assets/intranet/mtics/real-property/sres/process_manual_for_the_surplus_properties_program_-_feb_2020_-_version_2.pdf">Surplus Properties Program Process Manual</a>, the Strategic Real Estate Services Branch (SRES) has committed to proactively notifying all other Ministries, SUCH Sector Organization and Broader Public Sector (BPS) Entities of the availability of a new surplus property.</p>
+<p><b>Please forward this notification to any SUCH Sector Organization or BPS Entity that your ministry is responsible for to ensure any interest from Ministries or Agencies in the properties is identified.</b></p>
+<p><b>Should there be no interest in the property detailed below from your Ministry or any SUCH Sector Organization or BPS Entity that your Ministry is responsible for, please respond to confirm. </b></p>
+<p>This email is to advise that the following property has been identified as surplus by the current holder of the property and is available for redeployment if there is a need by your Ministry, SUCH Sector Organization or BPS Entity:</p>
+<p>
+    <ol>
+        @foreach (var property in Model.Project.Properties)
+        {
+        <li>
+            @if (property.PropertyType == PropertyTypes.Land)
+            {
+                var appraised = property.Parcel.Evaluations.OrderByDescending(e => e.Date).FirstOrDefault(e => e.Key == EvaluationKeys.Appraised);
```

Comment:
This value will never be populated on a property anymore.

Reply:
Same goes for all templates.

Reply:
I have another story to deal with this, but I suppose I can do both in this one migration.

Reply:
It isn't necessary

Reply:
@Fosol PA-2526(financials linked to properties not projects in erp notifications), PA-2527(new erp note type in erp notifications) and PA-1698 (do not default project financials to $0) are all tightly coupled, so I've update the review to contain fixes for all of those issues.